Portal/src/Datahub.Infrastructure/Datahub.Infrastructure.csproj

```diff
@@ -14,6 +14,9 @@
       <PackageReference Include="Azure.ResourceManager.Monitor" Version="1.3.0" />
       <PackageReference Include="Azure.ResourceManager.Storage" Version="1.2.0" />
       <PackageReference Include="Google.Cloud.Storage.V1" Version="4.6.0" />
+      <PackageReference Include="MassTransit" Version="8.2.2" />
+      <PackageReference Include="MassTransit.Abstractions" Version="8.2.2" />
```

Comment:
we can prob ignore the abstraction and mass transit packages

Reply:
weird, not sure how that made it in